content/docs/concepts/architecture/index.md

```diff
@@ -54,25 +55,26 @@ api, bridge, or resource-service.
 
 ### api-service
 
-The [Keptn API](../../0.18.x/reference/api/) provides a REST API
+The [Keptn API](../../0.19.x/reference/api/) provides a REST API
 that allows you to communicate with Keptn.
 It provides endpoints to authenticate, get metadata about the Keptn installation within the cluster,
-forward CloudEvents to the NATS cluster, and triggering evaluations for a service.
-
-The documentation shows you how to access the API and to explore the available endpoints.
+forward [CloudEvents](../../0.19.x/reference/miscellaneous/events)
+to the NATS cluster, and trigger evaluations for a service.
 
 ### mongodb-service
 
 The *mongodb-datastore* stores event data in a MongoDB deployed in your Keptn namespace.
 Hence, the service provides the REST endpoint `/events` to query events.
+The `mongodb-datastore` and `shipyard-controller` pods
+have direct connections to mongodb (keptn-mongo).
 
 ### resource-service
 
 The *resource-service* is a Keptn core component
 that manages resources for Keptn project-related entities, i.e., project, stage, and service.
 This replaces the `configuration-service` that was used in Keptn releases before 0.16.x.
-It uses the Git-based [upstream repository](../../0.18.x/manage/git_upstream)/ mounted as a persistent volume
-to store the resources with version control,
+It uses the Git-based [upstream repository](../../0.19.x/manage/git_upstream)
+which is mounted as a persistent volume to store the resources with version control,
```

Comment:
Nitpick: instead of a persistent volume, the resource service now hosts its files in an `emptyDir` volume, which means that a new pod of the service always starts with an empty volume

Reply:
Partially done.  Need to specify when emptyDir replaced the PVC.